### PR TITLE
Jobs,vendors,rigs

### PR DESCRIPTION
**Pull request by blackwolf602**
add: new sprites for mining and engineer rig
add: Jobs now roundstart with the proper ds clothing
add: ds clothing now added to all the proper vendor and cloth vendors
fix: mara lockers should now not spawn with all the contents of other lockers
no more felinism unless plasma man :)

### CODE REVIEW

File: _maps/map_files/Mara17/Mara17_Middle.dmm [hunk 1]
```diff
@@ -586,7 +586,7 @@
 /turf/open/floor/deadspace/random/golf_brown,
 /area/mara17/interior/medium/hydrophonics)
 "ce" = (
-/obj/structure/closet/secure_closet/security/mara,
+/obj/structure/closet/secure_closetmara/security,
```
Comment (DTraitor) [inline _maps/map_files/Mara17/Mara17_Middle.dmm:589]: Shouldn't it be `/obj/structure/closet/secure_closet/mara/security`?

File: deadspace/code/modules/weapons/handheld/melee.dm [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /obj/item/knife/unitology
-	name = "unitology_ritual_blade"
+	name = "Unitolgist ritual blade"
```
Comment (DTraitor) [inline deadspace/code/modules/weapons/handheld/melee.dm:2]: As far as I know most names start with lowercase letter

**Review by DTraitor — COMMENTED** [review]
Looks almost good though I want to discuss how we are going to handle custom jobs first

File: deadspace/code/modules/weapons/handheld/melee.dm (hunk 2 above)
Comment (blackwolf602) [reply deadspace/code/modules/weapons/handheld/melee.dm:2]: fixed

File: _maps/map_files/Mara17/Mara17_Middle.dmm (hunk 1 above)
Comment (blackwolf602) [reply _maps/map_files/Mara17/Mara17_Middle.dmm:589]: fixed

**Pull request merged**